### ISSUE TITLE
🐛 Page <title> missing from accessibility statement page

### ISSUE THREAD

**Issue by jemnery** [ministryofjustice/find-moj-data#1562]
### Describe the bug.

Ironically, an accessibility fail - the `<title>` should be "Accessibility statement - Find MoJ Data" (consistent with other pages).

<img width="1421" alt="Image" src="https://github.com/user-attachments/assets/77e75880-d927-4de7-96c7-e62883a76ca3" />

**Comment by mitchdawson1982** [comment]
[PR](https://github.com/ministryofjustice/find-moj-data/pull/1565/files)
